prime-router/docs/api/waters-delivery.yml

```diff
@@ -150,6 +150,41 @@ paths:
           description: unauthorized
         '404':
           description: report does not exist
+  /waters/report/{receiver}/facilities:
```

Comment:
@TurkeyFried should this be something like /waters/report/{org.receiver}/facilities
Since we need to pass the organization and the receiver?

Reply:
Yeah, that's a more accurate name

Reply:
Correction - organization and the sender